### ISSUE TITLE
End game screen / Player explosion

### ISSUE THREAD

**Issue by sudoryan** [seanjones2848/cpp_shooter#5]
Would be nice to show the player's ship exploding for a second or two and show an end game screen with their score and the high score. Right now it cuts straight to the menu at death.

**Comment by slothlysage** [comment]
yes, that would be the desired ending, it just cut to reload screen cause I was lazy, totes down to implement good death screen, the vestigial code for gameOver in Game class is where that would be held called in just after the game loop in main.
